src/index.tsx

```diff
@@ -19,6 +20,9 @@ import UserUpdater from './state/user/updater'
 import ThemeProvider, { FixedGlobalStyle, ThemedGlobalStyle } from './theme'
 import getLibrary from './utils/getLibrary'
 
+// Enables use of the useWhatChanged hook in dev environment
+setUseWhatChange(process.env.NODE_ENV === 'development')
```

Comment:
I'm always afraid that we make a typo on "development", should we make it a constant?